src/components/Router/index.js

```diff
@@ -209,25 +212,54 @@ class CrossDeviceMobileRouter extends Component {
     this.sendMessage('client success', { captures })
   }
 
+  isFallbackDisabledAndShouldUseCamera = (step) => {
+    const options = step.options
+    return (
+      options &&
+      options.uploadFallback === false &&
+      (step.type === 'face' || options.useLiveDocumentCapture)
```

Comment:
What about Doc Auto Capture?

Reply:
We don't have `uploadFallback` on doc auto capture, we always return to the uploader because is a beta feature